canvas.h

```diff
@@ -95,3 +93,26 @@ class Canvas : public ICanvas
     uint32_t _height;
     vector<unique_ptr<ILEDFeature>> _features;
 };
+
+
+#include "ledfeature.h"
```

Comment:
I have an include order dependency that means LEDFeature has to include Canvas and Canvas needs LEDFeature.  Having this include here is a dirty hack until I figure out a better way to address it.... hopefully